controllers/workspace/devworkspace_controller.go

```diff
@@ -372,6 +372,9 @@ func (r *DevWorkspaceReconciler) stopWorkspace(workspace *devworkspace.DevWorksp
 func (r *DevWorkspaceReconciler) syncTimingToCluster(
 	ctx context.Context, workspace *devworkspace.DevWorkspace, timingInfo map[string]string, reqLogger logr.Logger) {
 	if timing.IsEnabled() {
+		if workspace.Annotations == nil {
```

Comment:
Could you make check that we don't have another places where we try to write into workspace labels/annotations without nil check?

I just wonder if we won't get the same error on further steps if we disable timing.